server/middleware/frontEndComponents.ts

```diff
@@ -23,7 +25,14 @@ export default function getFrontendComponents({ componentService }: Services, us
       res.locals.feComponentsMeta = meta
 
       /* Set feature toggle for using Alerts API */
-      config.featureToggles.alertsApiEnabled = meta?.services?.some(service => service.id === 'alerts')
+      if ('activeCaseLoadId' in res.locals.user) {
+        await featureToggleService.setFeatureToggle(
```

Comment:
If this does have performance implications, then we could incorporate a scheduled task in to the feature toggle store to go and fetch it every minute or so.